arex-web-core/src/main/java/com/arextest/web/core/repository/mongo/ReportPlanItemStatisticRepositoryImpl.java

```diff
@@ -116,6 +116,29 @@ public boolean findAndModifyBaseInfo(PlanItemDto result) {
         return true;
     }
 
+    @Override
+    public boolean findAndModifyCaseMap(PlanItemDto result) {
+        if (result.getPlanItemId() == null) {
+            return false;
+        }
+        Update update = MongoHelper.getUpdate();
+        update.setOnInsert(DATA_CHANGE_CREATE_TIME, System.currentTimeMillis());
```

Comment:
" MongoHelper.getUpdate()" exists the setOnInsert operation of "DATA_CHANGE_CREATE_TIME"

Reply:
fixed